src/Table/TablePagination.spec.js

```diff
@@ -213,5 +213,35 @@ describe('<TablePagination />', () => {
       // now, the third page doesn't exist anymore
       assert.strictEqual(page, 1);
     });
+
+    it('should not display 0 as start number if the table is empty ', () => {
+      function ExampleTable(props) {
+        // setProps only works on the mounted root element, so wrap the table
+        return (
+          <table>
+            <TableFooter>
+              <TablePagination
+                count={0}
+                page={0}
+                onChangePage={noop}
+                onChangeRowsPerPage={noop}
+                {...props}
+              />
+            </TableFooter>
+          </table>
+        );
+      }
+
+      const wrapper = mount(<ExampleTable rowsPerPage={5} />);
```

Comment:
is shallow possible?

Reply:
Yeah, I should split the test. I'll update this.

Reply:
Unfortunately, I can't get the text of the nested `typography` element if the Table is shallow. :confused: I updated the test to actually test if #8431 is fixed.